src/nodes.coffee

```diff
@@ -1892,15 +1892,17 @@ exports.Try = class Try extends Base
   # is optional, the *catch* is not.
   compileNode: (o) ->
     o.indent  += TAB
+    scope     = o.scope
     tryPart   = @attempt.compileToFragments o, LEVEL_TOP
 
     catchPart = if @recovery
-      placeholder = new Literal '_error'
+      generatedErrorVariableName = scope.freeVariable '_error'
```

Comment:
Why not simply `o.scope.freeVariable` and remove `scope = o.scope` (which btw could be written `{scope} = o`) above?